app/views/scheduled_tests/_form.html.erb

```diff
@@ -0,0 +1,361 @@
+              <%= coll.name %>
+            </h5>
+          </div>
+
+          <table class="table table-striped table-bordered hidden-table"
+                 width="100%" border="0" cellspacing="0" cellpadding="0">
+            <thead>
+              <tr class="small-table-headings">
+                <th><input type="checkbox" class="select-all-checkbox"></th>
+                <th><%= model_class.human_attribute_name(:name) %></th>
+                <th><%= model_class.human_attribute_name(:description) %></th>
+                <th><%= model_class.human_attribute_name(:updated_at) %></th>
+              </tr>
+            </thead>
+            <tbody>
+              <% coll.testsets.each do |testset| %>
+                <tr class="testset scheduled-test-testset">
+                  <td class="testset-form-checkbox-column">
+                    <input type="checkbox" class="testset-checkbox collection<%= testset.collection.id %>"
+                        value="<%= testset.id %>"
+                         <%= "checked='checked'" if @scheduled_test.test_ids && @scheduled_test.test_ids.include?("#{testset.id}") %> />
+                  </td>
+                  <td class="testset-index-name-column cursor-hand" onclick="location.href='<%=collection_testset_path(coll, testset)%>'">
+                    <%= testset.name %>
+                  </td>
+                  <td class="testset-form-description-column description-font-size cursor-hand" onclick="location.href='<%=collection_testset_path(coll, testset)%>'">
+                    <%= testset.description %>
+                  </td>
+                  <td class="testset-form-action-updated-at-column cursor-hand" onclick="location.href='<%=collection_testset_path(coll, testset)%>'">
+                    <%=l testset.updated_at, format: :short %>
+                  </td>
+                </tr>
+              <% end %>
+            </tbody>
+          </table>
+          <div class="clearfix"></div>
+        </div>
+      <% end %>
+    </div>
+  </div>
+
+  <hr />
+
+  <div class="form-group">
+    <%= f.label :browser, :class => 'control-label col-md-2' %>
+    <div class="col-md-10">
+      <%= f.collection_select(:browser_ids, BrowserType.order('name ASC'), :id, :name,
+      {},
+      { :class => 'select-list multiselect form-control browser-checkbox',
+      :multiple => true }) %>
+    </div>
+  </div>
+  <div class="form-group">
+    <%= f.label :environment, :class => 'control-label col-md-2' %>
+    <div class="col-md-10">
+      <%= f.collection_select(:environment_id, Environment.where(user: current_user).order('name ASC'), :id, :name,
+      {},
+      { :class => 'select-list form-control environment-select-list' }) %>
+    </div>
+  </div>
+  <div class="form-group">
+    <%= f.label :next_test, :class => 'control-label col-md-2' %>
+    <div class="col-md-10">
+      <%= f.text_field :next_test, :class => 'form-control datepicker',
+                :id => "nextTestScheduledTest" %>
```

Comment:
The time here displays with the offset applied. In comparison the time displayed on the schedule tests index page does not. From a users perspective I would prefer all of the times displayed to me throughout the UI have my local time offset applied.